lib/dmv_factory.rb

```diff
@@ -0,0 +1,41 @@
+class DmvFactory
+
+  def create_colorado_facilities(buildings)
+    dmv_buildings = []
+    buildings.map do |building|
+      facility_info = {}
+      facility_info[:id] = building[:dmv_id]
+      facility_info[:address] = building[:address_li]
+      facility_info[:zip] = building[:zip]
+      facility_info[:phone] = building[:phone]
+      dmv_buildings << facility_info
+      # DmvFactory.new(facility_info)
+    end
+    dmv_buildings
```

Comment:
Hmm... what is the return value of this method now? It looks like we just have an array of hashes, but we wanted to return an array of Facility objects. We need one more step here, on line 11, we need this to say `dmv_buildings << Facility.new(facility_info)` so that we're creating Facility objects. Otherwise this looks good. 
Also, don't need to use .map if we're using the accumulator. Or use no accumulator and .map